Src/Clients/Core/WebAPI/Controllers/ActorsController.cs

```diff
@@ -38,12 +38,12 @@ public ActionResult<IQueryable<ActorLookupDto>> GetAll()
         public ActionResult<ActorLookupDto> Get(int id)
         {
             var result = _repository.Find(e => e.ActorId == id).FirstOrDefault();
-            //result.Photos = _actorPhotosRepository.Find(e => e.ActorId == id);
+            var photoResult = _actorPhotosRepository.Find(e => e.ActorId == id).ToList();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'photoResult'.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5384135)